.github/workflows/trigger-downstream-flake-update.yml

```diff
@@ -16,13 +16,38 @@ jobs:
         include:
           - downstream-repo: 'blocksense-network/infra'
     steps:
+      - name: Compare nix outputs
+        id: compare
+        run: |
+          # Define your command here (replace this example)
+          CMD="nix eval --raw .\#nixosConfigurations.example-setup-01.config.system.build.toplevel.outPath1"
```

Comment:
```suggestion
          function get_machine_outpath {
            commit="$1"
            config="$2"
            nix eval --raw "github:blocksense-network/blocksense?rev=${commit}#nixosConfigurations.${config}.config.system.build.toplevel.outPath1"
          }
```

Reply:
On second thought, can you implement my suggestion as a `just` recipe, so it could be used outside the CI?